allennlp/service/server_sanic.py

```diff
@@ -3,21 +3,21 @@
 
 from sanic import Sanic, response, request
 from sanic.exceptions import ServerError
+from typing import Dict
 
-DEFAULT_ARCHIVE_FILES = {
-        'machine-comprehension': 'tests/fixtures/bidaf/serialization/model.tar.gz',
-        'semantic-role-labeling': 'tests/fixtures/srl/serialization/model.tar.gz',
+default_config = {
+        'machine-comprehension': 'https://s3-us-west-2.amazonaws.com/allennlp/models/bidaf-model-2017.08.26.tar.gz',
+        'semantic-role-labeling': 'https://s3-us-west-2.amazonaws.com/allennlp/models/srl-model-2017.08.28.tar.gz',
```

Comment:
I don't know if you care, but I think you also do urls like

`https://allennlp.s3.amazonaws.com/models/...`